### PR TITLE
autotest: fix un-set variable problem in verify_innov

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by peterbarker** [comment]
Message ordering can lead to a "alt is not set" error

File: Tools/autotest/arducopter.py [hunk 1]
```diff
@@ -2419,6 +2419,7 @@ def EK3_RNG_USE_HGT(self):
         self.reboot_sitl()
 
         # add a listener that verifies rangefinder innovations look good
+        global alt
```
Comment (rmackay9) [inline Tools/autotest/arducopter.py:2422]: I wonder if "global" is a bit too wide a scope?

Comment (peterbarker) [reply Tools/autotest/arducopter.py:2422]: Honestly don't know how this scoping works, and how to get a tighter one.  I do in Perl, but that's not going to help here ;-)

This is a pattern we use elsewhere, however.  So if we do work out how to fix it, we can do it in lots of places :-)

Comment (rmackay9) [reply Tools/autotest/arducopter.py:2422]: OK!  fine with me.

**Pull request merged**